pycalphad/core/workspace.py

```diff
@@ -254,25 +270,40 @@ class Workspace:
     database: Database = TypedField(lambda _: None)
     components: SpeciesList = ComponentsField(depends_on=['database'])
     phases: PhaseList = PhasesField(depends_on=['database', 'components'])
-    conditions: Conditions = ConditionsField(lambda wks: Conditions(wks))
+    conditions: Conditions = ConditionsField(lambda wks: Conditions(wks), depends_on=['components', 'models'])
```

Comment:
where does the conditions dependence on models come from?

Reply:
https://github.com/pycalphad/pycalphad/pull/554/files/bc95d629328445e8b611778bc2e4c0449651a05f#diff-c8bfc00897c8c4e08c283c9f9319933a84b8d6a356d1f98f3704aa40d05ec2d2R130